dashboard/app/views/levels/_blockly.html.haml

```diff
@@ -81,18 +52,19 @@
       $('.cdo-qtips').qtip('hide');
     });
     // See assets/javascripts/sendReport for definition of lastServerResponse
-    var appOptions = {
-      levelId: '#{@level.level_num}',
-      #{ "level_source_id: #{@level_source_id}," if @level_source_id }
-      #{ "send_to_phone_url: '#{@phone_share_url}'," if @phone_share_url }
-      #{ "sendToPhone: true," if send_to_phone}
-      #{ "disableSocialShare: true," if (@current_user && @current_user.under_13?) || @embed}
-      #{ "embed: true," if embed}
-      #{ "scriptId: '#{@script.id}'," if @script }
-      #{ "levelGameName: '#{@level.game.name}'," if @level.game }
-      level: level,
-      #{ "skinId: '#{@level.skin}'," if @level.is_a?(Blockly) }
-      callouts: #{@callouts.to_json} || [],
+    var appOptions = #{app_options.to_json};
+    var baseOptions = {
+      locale: '#{js_locale}',
+      containerId: 'blocklyApp',
+      Dialog: Dialog,
```

Comment:
It's existing code -- good call to change cdoSounds -> CDOSounds but prefer to do this after the refactor.